readthedocs/core/symlink.py

```diff
@@ -198,7 +198,13 @@ def symlink_subprojects(self):
                 symlink_dir = os.sep.join(symlink.split(os.path.sep)[:-1])
                 if not os.path.lexists(symlink_dir):
                     safe_makedirs(symlink_dir)
-                run('ln -nsf %s %s' % (docs_dir, symlink))
+                # TODO this should use os.symlink, not a call to shell. For now,
+                # this passes command as a list to be explicit about escaping
+                # spaces.
+                status, _, stderr = run(['ln', '-nsf', docs_dir, symlink])
```

Comment:
All of the other calls in this file seem to not pass a list here. Should those be fixed, too?